ctrl:
  ros__parameters:
    Kp: 5.5
    Ki: 5.0
    Kd: 0.02
    sample_time: 0.02

dc_motor:
  ros__parameters:
    sample_time: 0.02
    sys_gain_K: 1.75
    sys_tau_T: 0.5
    initial_conditions: 0.0

set_point_node:
  ros__parameters:
    amplitude: 2.0
    omega: 1.0
    signal_type: "sine"  # Options: 'sine', 'square', 'step'